README.md

```diff
@@ -40,3 +40,42 @@ This repo contains a UML sequence diagram, which can be updated as follows:
 1. Run `docker pull think/plantuml` to download [this useful PlantUML image](https://hub.docker.com/r/think/plantuml/)
 2. Edit `./sequenceDiagram/source.uml` as needed
 3. Run `cat sequenceDiagram/source.uml | docker run --rm -i think/plantuml -tpng > sequenceDiagram/source.png` to output
+
+## Workflow
+
+The process of converting and sending a document is handled by a number of interrelated services:
+
+### Fetch task (run once a minute)
+
+- Fetch all new documents in the print queue from comino (compare with latest document in Dynamo)
+- Store in Dynamo (state = `Fetched`)
+- Push into SQS
+
+### Conversion task
+
+- Fetch document ID from SQS
+- Look up document in Comino and fetch from documents API
```

Comment:
Comino isn't involved here, We get the doc details from the local db then get the document from the documents API